graylog2-server/src/main/java/org/graylog2/shared/security/RestrictToMasterFilter.java

```diff
@@ -32,7 +32,7 @@
 
     @Override
     public void filter(ContainerRequestContext requestContext) throws IOException {
-        LOG.warn("Rejected request that is only allowed against master nodes. Returning HTTP 403.");
+        LOG.warn("Rejected request to <{}> which is only allowed against master nodes.", requestContext.getUriInfo().getPath());
```

Comment:
Can `requestContext.getUriInfo()` be `null` under any circumstances?

Reply:
This gets initialized in jersey's `ConterRequest` constructor. So I'd say this never happens.